src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpAndCollectorStep.java

```diff
@@ -35,7 +40,7 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
         return true;
     }
 
-    protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(String saeFullyQualifiedFileName) {
+    protected boolean processCurrentFileAndExtractPdpFeedAndCollectorFromSAEFile(String saeFullyQualifiedFileName, ConcurStandardAccountingExtractBatchReportData reportData) {
```

Comment:
I don't think it needs to be changed now, but when you start working on building the reports, we will want to update the log statements to use the new function name